packages/nimble-components/src/banner/styles.ts

```diff
@@ -33,6 +33,7 @@ export const styles = css`
         font-size: 12.8px;
         align-items: top;
         overflow: hidden;
+        word-break: break-word;
```

Comment:
mdn says `word-break: break-word` is deprecated: https://developer.mozilla.org/en-US/docs/Web/CSS/word-break
Looks like `overflow-wrap` should be configured: https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#break-word

Reply:
I think I had meant to use word-wrap rather than word-break, though I believe it's essentially equivalent to word-break/overflow-wrap. Confusing! I'll double-check.

Reply:
Actually, this is the way I intended. `word-wrap` is deprecated, not `word-break`. From what I can tell, `word-break: break-word` and `overflow-wrap: break-word` are essentially equivalent. Maybe `overflow-wrap` is more common/recognized, though, so I can use that instead.

Reply:
> word-wrap is deprecated, not word-break.

The setting used in the PR, `word-break: break-word`, is what the MDN docs I linked is saying is deprecated:

![image](https://github.com/ni/nimble/assets/1588923/f80b928e-4119-4516-aaea-a7676a394601)

Are the MDN docs incorrect? Any source for why that is not expected to be deprecated?

Reply:
Oh, I didn't see that the `break-word` value was deprecated. It didn't say so in the [doc for that value](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#break-word). I searched a little for other confirmation that the `break-word` value is deprecated, and [caniuse](https://caniuse.com/mdn-css_properties_word-break_break-word) seems to back that up. Anyway, I had already switched to using `overflow-wrap: break-word`, so we should be good.

Reply:
The comment was resolved and I was reset but I don't see a change?

Reply:
🤦‍♂️Duh, I hadn't pushed the commit. Sorry.

Reply:
Reminder to update the PR description too :-)